src/screens/DiscoveryScreenV2/fragment/DomainFragment.js

```diff
@@ -204,15 +226,13 @@ const DomainFragment = ({
     if (item.separator) {
       return (
         <>
-          {renderRecentSearch(index)}
           <DiscoveryTitleSeparator text="Suggested Domains" key="domain-title-separator" />
         </>
       );
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The condition `if (item.separator)` checks if the item is a separator. If it is, it renders a DiscoveryTitleSeparator component. This is a good way to handle different types of items in a list.
```javascript
if (item.separator) {
  return (
    <>
      <DiscoveryTitleSeparator text="Suggested Domains" key="domain-title-separator" />
    </>
  );
}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->